source/shlobj.py

```diff
@@ -1,22 +1,32 @@
+# -*- coding: UTF-8 -*-
 #shlobj.py
 #A part of NonVisual Desktop Access (NVDA)
 #Copyright (C) 2006-2017 NV Access Limited, Babbage B.V.
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
 
+"""
+This module wraps the SHGetFolderPath function in shell32.dll and defines the necessary contstants. CSIDL (constant special item ID list) values provide a unique system-independent way to identify special folders used frequently by applications, but which may not have the same name or location on any given system. For example, the system folder may be "C:\Windows" on one system and "C:\Winnt" on another.

```

Comment:
Could you please split this onto multiple lines, around the 80 - 100 character point?